src/lib/seam/components/DeviceDetails/ThermostatDeviceDetails.tsx

```diff
@@ -38,54 +33,15 @@ interface ThermostatDeviceDetailsProps
 
 export function ThermostatDeviceDetails({
   device,
-  errorFilter,
-  warningFilter,
-  disableLockUnlock,
-  disableCreateAccessCode,
-  disableEditAccessCode,
-  disableDeleteAccessCode,
   disableResourceIds,
   disableConnectedAccountInformation,
-  disableClimateSettingSchedules,
```

Comment:
We should removed this from the API. Are there any other references to ClimateSettingSchedules in this project?

Reply:
Yep. Removed all in https://github.com/seamapi/react/pull/659/commits/3457ca40076abd9cdd9c7840a6ec6c0502769d28